cypress/e2e/signIn.cy.js

```diff
@@ -28,7 +32,27 @@ describe('Sign In page', () => {
     homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it('should not provide an ability to log in with wrong email', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    signInPage.typeEmail(wrongUser.email);
+    signInPage.typePassword(user.password);
+    signInPage.clickSignInBtn();
+
+    signInPage.verifyEmailErorr();
+
+  });
+
+  it('should not provide an ability to log in with wrong password', () => {
+
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
+
+    signInPage.typeEmail(user.email);
+    signInPage.typePassword(wrongUser.password);
+    signInPage.clickSignInBtn();
 
+    signInPage.verifyPasswordErorr();
```

Comment:
There seems to be a typo in the method name `verifyPasswordErorr`. It should likely be `verifyPasswordError` to correctly reflect the purpose of verifying a password error.